packages/aila/tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "moduleResolution": "bundler",
     "esModuleInterop": true,
     "types": ["jest"],
-    "typeRoots": ["./../../node_modules/@types", "./src/types"],
+    "typeRoots": [
+      "./../../node_modules/@types",
+      "./src/types",
+      "./src/utils/language/*.d.ts"
```

Comment:
Remove the direct import of the d.ts file for americanisms - this was causing a turbopack issue